packages/vaadin-lumo-styles/src/components/crud-edit.css

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+:host {
+  display: block;
+  min-width: auto;
+  margin: 0;
+  font-family: 'lumo-icons', var(--lumo-font-family);
```

Comment:
The fallback to `var(--lumo-font-family)` seems unnecessary here. We can keep it as the original CSS has it.

Reply:
Yeah, this shows up in other components too, so it would make sense to address this separately (if we decide so).